.github/workflows/cppcheck-monitor.yaml

```
@@ -1,13 +1,12 @@
-name: cppcheck-all
+name: cppcheck-monitor
```

```suggestion
name: cppcheck-daily
```

This way the scope, schedule and function will be more clear, what do you think?

---

Agreed, thanks for the proposal.